diagnostic.md

```diff
@@ -49,7 +49,7 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+$cornflower: tint($cornflower, 20%);
```

Comment:
Not only is this not correct, it could cause a problem - you'd be redefining `$cornflower` at some point within your CSS, so that some references would be darker and some would be lighter.